proto/internalapi/central/sensor_events.proto

```diff
@@ -92,7 +92,9 @@ message SensorEvent {
         storage.ComplianceOperatorScan               compliance_operator_scan                 = 24;
 
```

Comment:
why does tag go from 24 to 27?

Reply:
It doesn't.  they just are not in order.